person.rb

```diff
@@ -0,0 +1,40 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @id = id
+    @parent_permission = parent_permission
+  end
+
+  # getters
+  def gets_id
+    @id
+  end
+
+  def gets_name
+    @name
+  end
+
+  def gets_age
+    @age
+  end
+
+  # setters
+  def sets_age(age)
+    @age = age
+  end
+
```

Comment:
- [Optional] Dear @isume295, for the `geter` and `setter`  ruby has a shorthand form and can be achieved by using `atrr_reader, attr_writer, attr_accessor`. Consider looking through [this document](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more.

Reply:
okay I will look through the documentation, Thank you